frontend/reducers/projects_reducer.js

```diff
@@ -2,10 +2,14 @@ import { RECEIVE_PROJECT, RECEIVE_PROJECTS } from "../actions/project_actions";
 
 const projectsReducer = (oldState = {}, action) => {
   Object.freeze(oldState);
-
+  let nextState = {};
   switch(action.type) {
     case RECEIVE_PROJECTS:
-      return Object.assign({}, oldState, action.projects);
+      const projectsArray = Object.values(action.projects);
```

Comment:
I needed to turn the projects object into an array in order to iterate over it. I'm not totally sure that this was the right solution here.